pgmpy/inference/ExactInference.py

```diff
@@ -3,7 +3,7 @@
 import numpy as np
 from pgmpy.inference import Inference
 from pgmpy.factors.Factor import factor_product
-
+import networkx as nx
```

Comment:
@pratyakshs Have a look at PEP8 (https://www.python.org/dev/peps/pep-0008/#imports) regarding the order of imports in a file.